src/views/v2/list/event/featured-image.php

```diff
@@ -9,30 +9,28 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.3
+ * @version TBD
  *
  */
-$event    = $this->get( 'event' );
-$event_id = $event->ID;
 
-if ( ! has_post_thumbnail( $event_id ) ) {
+if ( empty( $event->thumbnail->large->url ) ) {
```

Comment:
Use the `exist` property of the event thumbnail here:
```suggestion
if ( ! $event->thumbnail->exists ) {
```